tdrs-frontend/src/components/Header/Header.jsx

```diff
@@ -4,6 +4,7 @@ import { useSelector } from 'react-redux'
 import closeIcon from 'uswds/dist/img/close.svg'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faSignOutAlt, faUserCircle } from '@fortawesome/free-solid-svg-icons'
+import NavItem from '../NavItem/NavItem'
```

Comment:
I created a new component called `NavItem` that will allow us to render a new navigation tab by passing the pathname, tabTitle, and href.